### PR TITLE
Show the result of post in post intent page

### PR DESCRIPTION
**Pull request by akihikodaki**

### CODE REVIEW

**Comment by akihikodaki** [comment]
046ffb0b41befada830422a397a4100e6cdfeb92
screenshot-2018-1-30 mastodon dev

File: app/javascript/mastodon/containers/compose_container.js [hunk 1]
```diff
@@ -29,7 +30,9 @@ export default class TimelineContainer extends React.PureComponent {
     return (
       <IntlProvider locale={locale} messages={messages}>
         <Provider store={store}>
-          <Compose />
+          <BrowserRouter basename='/web' forceRefresh>
+            <Compose />
+          </BrowserRouter>
```
Comment (ThibG) [inline app/javascript/mastodon/containers/compose_container.js]: What is this for?

File: app/javascript/mastodon/reducers/compose.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ const initialState = ImmutableMap({
   default_sensitive: false,
   resetFileKey: Math.floor((Math.random() * 0x10000)),
   idempotencyKey: null,
+  last: null,
```
Comment (ThibG) [inline app/javascript/mastodon/reducers/compose.js]: I'm not sure about having this information here, it seems to be only ever useful to the particular case of the web intent page… there ought to be a better way, although I'm not familiar enough with React to know better :/

File: app/javascript/mastodon/containers/compose_container.js (hunk 1 above)
Comment (akihikodaki) [reply app/javascript/mastodon/containers/compose_container.js]: For `Link` of React Router DOM. It does no routing, but it just navigates to the linked page.

File: app/javascript/mastodon/reducers/compose.js (hunk 2 above)
Comment (akihikodaki) [reply app/javascript/mastodon/reducers/compose.js]: I don't think there is a better way for this particular case. There are several options such as to have different Redux store or to utilize callbacks and React state. However they bring too much complexity for just one variable.

**Pull request closed without merging**